### PR TITLE
docs: use internal links to markdown files instead of urls

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

To avoid broken links due to (potential) changes in the repo URL in the future, switch all the links from url to local markdown file reference.

This makes also the markdown file easier to read.

### CODE REVIEW

File: LSPs/LSP-0-ERC725Account.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ The ERC725 general key value store allows for the ability to add any kind of inf
 
 ## Implementation
 
-A implementation can be found in the [ERC725Alliance/ERC725](https://github.com/ERC725Alliance/ERC725/blob/main/implementations/contracts/ERC725Account.sol) repository;
+An implementation can be found in the [lukso-network/lsp-smart-contracts](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP0ERC725Account/LSP0ERC725AccountCore.sol) repository.
```
Comment (Hugoo) [inline LSPs/LSP-0-ERC725Account.md:88]: You want to link to the develop branch ?

**Review by YamenMerhi — APPROVED** [review]

File: LSPs/LSP-5-ReceivedAssets.md [hunk 2]
```diff
@@ -21,10 +21,10 @@ Two keys are proposed to reference received asset smart contracts.
   - the index in the former array where the received asset address is stored.
   - an [ERC165 interface ID](https://eips.ethereum.org/EIPS/eip-165) to easily identify the standard used by each asset smart contracts, without the need to query the contracts directly. 
 
-The key `LSP5ReceivedAssetsMap` also helps to prevent adding duplications to the array, when automatically added via smart contract (e.g. a [LSP1-UniversalReceiverDelegate](https://github.com/lukso-network/LIPs/blob/master/LSPs/LSP-1-UniversalReceiver.md)).
+The key `LSP5ReceivedAssetsMap` also helps to prevent adding duplications to the array, when automatically added via smart contract (e.g. via an [LSP1-UniversalReceiverDelegate](./LSP-1-UniversalReceiver.md)).
```
Comment (Hugoo) [inline LSPs/LSP-5-ReceivedAssets.md:24]: Duplicates

File: LSPs/LSP-6-KeyManager.md [hunk 3]
```diff
@@ -34,26 +34,26 @@ The flow of a transactions is as follows:
 
 ## Motivation
 
-The benefit of a KeyManager is to externalise permission logic from [ERC725Y and X](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md) contracts (such as an [ERC725Account](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md)). This allows for such a logic to be upgraded without needing to change the core [ERC725Account](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md) contract.
+The benefit of a KeyManager is to externalise permission logic from [ERC725Y and X](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md) contracts (such as an [ERC725Account](./LSP-0-ERC725Account.md)). This allows for such an logic to be upgraded without needing to change the core [ERC725Account](./LSP-0-ERC725Account.md) contract.
```
Comment (Hugoo) [inline LSPs/LSP-6-KeyManager.md:37]: A logic

File: LSPs/LSP-6-KeyManager.md [hunk 4]
```diff
@@ -34,26 +34,26 @@ The flow of a transactions is as follows:
...
 The following ERC725Y keys are used to read permissions of certain addresses.
-These keys are based on the [LSP2-ERC725YJSONSchema](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md) standard, and use the key type **[Bytes20MappingWithGrouping](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#bytes20mappingwithgrouping)**
+These keys are based on the [LSP2-ERC725YJSONSchema](./LSP-2-ERC725YJSONSchema.md) standard, and use the key type **[Bytes20MappingWithGrouping](./LSP-2-ERC725YJSONSchema.md#bytes20mappingwithgrouping)**
 
 
 #### AddressPermissions[]
 
 Contains an array of addresses, that have some permission set.
-This is mainly useful for interfaces to know which address holds permissions.
+This is mainly useful for interfaces to know which address hold permissions.
```
Comment (Hugoo) [inline LSPs/LSP-6-KeyManager.md:56]: Holds

File: LSPs/LSP-6-KeyManager.md [hunk 5]
```diff
@@ -65,11 +65,13 @@ This is mainly useful for interfaces to know which address holds permissions.
 }
 ```
 
-For more info about how to access each index of the `AddressPermissions[]` array, see: [ERC725Y JSON Schema > `keyType` `Array`](https://github.com/lukso-network/LIPs/blob/master/LSPs/LSP-2-ERC725YJSONSchema.md#array)
+For more infos about how to access each index of the `AddressPermissions[]` array, see: [ERC725Y JSON Schema > `keyType` `Array`](./LSP-2-ERC725YJSONSchema.md#array)
```
Comment (Hugoo) [inline LSPs/LSP-6-KeyManager.md:68]: information

File: LSPs/LSP-6-KeyManager.md [hunk 6]
```diff
@@ -83,8 +85,8 @@ Contains [the permissions](#permission-values-in-addresspermissionspermissionsad
     
 #### AddressPermissions:AllowedAddresses:\<address\>
 
-Contains an array of allowed address a controlling address is allowed to interact with.
-IF no addresses are set, interacting with ALL addresses is allowed. IF one or more addresses is set, the controlling address, is only allowed to interact with those addresses.
+Contains an array of address (Externally Owned Accounts or smart contracts) a controlling address is allowed to interact with.
```
Comment (Hugoo) [inline LSPs/LSP-6-KeyManager.md:88]: addresses

File: LSPs/LSP-6-KeyManager.md [hunk 7]
```diff
@@ -83,8 +85,8 @@ Contains [the permissions](#permission-values-in-addresspermissionspermissionsad
...
+IF no addresses are set, interacting with ANY addresses is allowed. IF one or more addresses is set, the controlling address is restricted to interact only with these addresses.
```
Comment (Hugoo) [inline LSPs/LSP-6-KeyManager.md:89]:
```suggestion
- IF **no addresses** are set, interacting with ANY addresses is allowed.
- IF **one or more addresses** are set, the controlling address is restricted to interact only with these addresses.
```

File: LSPs/LSP-6-KeyManager.md [hunk 8]
```diff
@@ -98,7 +100,9 @@ IF no addresses are set, interacting with ALL addresses is allowed. IF one or mo
 
 #### AddressPermissions:AllowedFunctions:\<address\>
 
-Contains an array of bytes4 function signatures, the controlling address is allowed to call on other smart contracts.
+Contains an array of [bytes4 function signatures](https://docs.soliditylang.org/en/v0.5.3/abi-spec.html?highlight=selector#function-selector), the controlling address is allowed to call on other smart contracts.
+
+This permission acts as a restriction mechanism for an interacting with other smart contracts via an ERC725Account. It enables to limit which functions an address can call on a external smart contracts via the ERC725Account.
```
Comment (Hugoo) [inline LSPs/LSP-6-KeyManager.md:105]: An interacting?

File: LSPs/LSP-6-KeyManager.md [hunk 9]
```diff
@@ -170,7 +187,7 @@ SIGN               = 0x000000000000000000000000000000000000000000000000000000000
 function getNonce(address _address, uint256 _channel) public view returns (uint256)
 ```
 
-Returns the nonce that needs to be signed by an allowed key to be passed into the [executeRelayCall](#executeRelayCall) function. A signer can choose his channel number arbitrarily.
+Returns the nonce that needs to be signed by a allowed key to be passed into the [executeRelayCall](#executeRelayCall) function. A signer can choose his channel number arbitrarily.
```
Comment (Hugoo) [inline LSPs/LSP-6-KeyManager.md:190]:
```suggestion
Returns the nonce that needs to be signed by an allowed key to be passed into the [executeRelayCall](#executeRelayCall) function. A signer can choose his channel number arbitrarily.
```

File: LSPs/LSP-0-ERC725Account.md (hunk 1 above)
Comment (CJ42) [reply LSPs/LSP-0-ERC725Account.md:88]: yes

**Pull request merged**